translations/strings.yaml

```diff
@@ -690,6 +690,9 @@ inAppMessaging:
     comment: A badge shown in a message signifying that this message contains a survey
 
 global:
+  rtl:
+    value: ltr
+    comment: Write 'ltr' (left-to-right) or 'rtl' (right-to-left) to inform the app about the layout direction to use.
```

Comment:
Based on experience with Firefox, this is very brittle.

Can we check if QT provides info on the locale and its text direction? That's information included in intl libraries like ICU.